src/Nox.Reference.Data.World/Generated/Currencies.cs

```diff
@@ -1,155 +1,153 @@
-    UgandanShilling,
-    USDollar,
-    Uruguayanpeso,
-    Uzbekistanisom,
-    Venezuelanbolivar,
-    VietnameseDong,
-    NiVanuatuVatu,
-    SamoanTālā,
-    EastCaribbeandollar,
-    CFAFranc,
-    CFPFranc,
-    YemeniRial,
-    SouthAfricanRand,
-    ZambianKwacha,
-}
\ No newline at end of file
+public enum Currencies
+{
+	EmiratiDirham,
+	AfghanAfghani,
+	Albanianlek,
+	Armeniandram,
+	DutchGuilder,
+	AngolanKwanza,
+	Argentinepeso,
+	AustralianDollar,
+	Arubinflorin,
+	Azerbaijanimanat,
+	BosnianConvertibleMarka,
+	Barbadiandollar,
+	BangladeshiTaka,
+	Bulgarianlev,
+	BahrainiDinar,
+	BurundianFranc,
+	Bermudiandollar,
+	BruneianDollar,
+	BolivianBoliviano,
+	Brazilianreal,
+	Bahamiandollar,
+	BhutaneseNgultrum,
+	BotswanaPula,
+	Belarusianruble,
+	Belizedollar,
+	CanadianDollar,
+	SwissFranc,
+	ChileanPeso,
+	Yuanorchineserenminbi,
+	Colombianpeso,
+	CostaRicancolón,
+	Cubanconvertiblepeso,
+	Cubanpeso,
+	CapeVerdeanEscudo,
+	Czechkoruna,
+	DjiboutianFranc,
+	Danishkrone,
+	Dominicanpeso,
+	AlgerianDinar,
+	EgyptianPound,
+	Eritreannakfa,
+	EthiopianBirr,
+	Euro,
+	Fijiandollar,
+	FalklandIslandPound,
+	BritishPound,
+	Georgianlari,
+	GhanaianCedi,
+	Gibraltarpound,
+	Gambiandalasi,
+	GuineanFranc,
+	GuatemalanQuetzal,
+	Guyanesedollar,
+	HongKongdollar,
+	Honduranlempira,
+	Croatiankuna,
+	Haitiangourde,
+	Hungarianforint,
+	IndonesianRupiah,
+	IsraeliShekel,
+	IndianRupee,
+	IraqiDinar,
+	IranianRial,
+	IcelandicKrona,
+	Jamaicandollar,
+	JordanianDinar,
+	Japaneseyen,
+	KenyanShilling,
+	Kyrgyzstanisom,
+	CambodianRiel,
+	ComoranFranc,
+	NorthKoreanwon,
+	SouthKoreanwon,
+	KuwaitiDinar,
+	CaymanianDollar,
+	Kazakhstanitenge,
+	LaoorLaotianKip,
+	LebanesePound,
+	SriLankanRupee,
+	LiberianDollar,
+	Lesotholoti,
+	LibyanDinar,
+	MoroccanDirham,
+	MoldovanLeu,
+	MalagasyAriary,
+	MacedonianDenar,
+	BurmeseKyat,
+	MongolianTughrik,
+	MacauPataca,
+	MauritanianOuguiya,
+	Mauritianrupee,
+	MaldivianRufiyaa,
+	MalawianKwacha,
+	MexicoPeso,
+	MalaysianRinggit,
+	MozambicanMetical,
+	NamibianDollar,
+	NigerianNaira,
+	Nicaraguancórdoba,
+	Norwegiankrone,
+	NepaleseRupee,
+	NewZealandDollar,
+	OmaniRial,
+	Balboapanamérn,
+	Peruviannuevosol,
+	PapuaNewGuineanKina,
+	PhilippinePeso,
+	PakistaniRupee,
+	Polishzłoty,
+	Paraguayanguarani,
+	QatariRiyal,
+	Romanianleu,
+	SerbianDinar,
+	RussianRouble,
+	Rwandanfranc,
+	SaudiArabianRiyal,
+	SolomonIslanderDollar,
+	SeychelloisRupee,
+	SudanesePound,
+	Swedishkrona,
+	SingaporeDollar,
+	SierraLeoneanLeone,
+	SomaliShilling,
+	Surinamesedollar,
+	SyrianPound,
+	SwaziLilangeni,
+	ThaiBaht,
+	Tajikistanisomoni,
+	Turkmenistanmanat,
+	TunisianDinar,
+	TonganPaanga,
+	TurkishLira,
+	Trinidadiandollar,
+	TaiwanNewDollar,
+	TanzanianShilling,
+	UkrainianHryvnia,
+	UgandanShilling,
+	USDollar,
+	Uruguayanpeso,
+	Uzbekistanisom,
+	Venezuelanbolivar,
+	VietnameseDong,
+	NiVanuatuVatu,
+	SamoanTālā,
+	EastCaribbeandollar,
+	CFAFranc,
+	CFPFranc,
+	YemeniRial,
+	SouthAfricanRand,
+	ZambianKwacha,
}
```

Comment:
Why this changes each pr? Why it has comma in the end? I think we have a problem

Reply:
It should be excluded from solution so far. I added enum generator which is invoked during seeder.